packages/fbjs/src/key-mirror/keyMirror.js

```diff
@@ -32,7 +33,7 @@ var invariant = require('invariant');
  * @param {object} obj
  * @return {object}
  */
-var keyMirror = function(obj) {
+var keyMirror = function<T: {}>(obj: T): $ObjMapi<T, <K>(K) => K> {
```

Comment:
This change is mostly in www already, difference is `Object` vs `{}` and `O` vs `T`. Can you pick one and update the other? https://fburl.com/otx6xumj

(I need to do a full re-sync at some point soon)